config/config.cfg

```diff
@@ -95,7 +102,7 @@ entities = (
                     BACKWARD = "Q",
                     LEFT = "Z",
                     RIGHT = "S",
-                    ACTION1 = "LEFT_CLICK",
+                    ACTION1 = "SPACE",
```

Comment:
We do but it doesn't matter :) It will do both actions! Since only the first action is mapped for the moment, it does nothing for the second action ^^